nall/nall/decode/cue.hpp

```diff
@@ -178,7 +180,8 @@ inline auto CUE::loadIndex(std::vector<string>& lines, u32& offset) -> Index {
   Index index;
 
   lines[offset].itrimLeft("INDEX ", 1L);
-  string sector = lines[offset].split(" ").last().strip();
+  auto parts3 = nall::split(lines[offset], " ");
```

Comment:
Suffixed variable again unnecessary.
```suggestion
  auto parts = nall::split(lines[offset], " ");
  string sector = parts.empty() ? string{} : parts.back().strip();
```